php/datasource/class-fieldmanager-datasource.php

```diff
@@ -78,16 +104,16 @@ public function __construct( $options = array() ) {
 			if ( Fieldmanager_Field::$debug ) {
 				throw new FM_Developer_Exception( $message );
 			} else {
-				wp_die( $message, esc_html__( 'Invalid Datasource Options', 'fieldmanager' ) );
+				wp_die( $message, esc_html__( 'Invalid Datasource Options', 'fieldmanager' ) ); // WPCS: XSS ok.
```

Comment:
Escaping was added to `wp_die()` elsewhere in the PR. Could it be added here so we don't need the `// WPCS` exception?